FakebookNotifications/FakebookNotifications.Domains/Interfaces/INotificationsRepo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FakebookNotifications.Domains.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FakebookNotifications.DataAccess.Models.Interfaces
+{ 
+    public interface INotificationsRepo
+    {
+        /// <summary>
+        /// Get all notifications from each user.
+        /// </summary>
+        /// <returns></returns>
+        Task<IEnumerable<Notification>> GetAllNotifications();
+        /// <summary>
+        ///  Get All Notifications by a person's email
+        /// </summary>
+        /// <returns></returns>
+        Task<IEnumerable<Notification>> GetNotificationsByEmail(Notification notification);
+        /// <summary>
+        ///  Add a Notification to the databse
+        /// </summary>
+        /// <returns> A bool that let's us know if we passed or failed</returns>
```

Comment:
documentation is clumsy. fix typos, document all the return values and params.
https://www.mking.net/blog/documenting-task-returns